doa_estimation/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(doa_estimation)
+
+find_package(catkin REQUIRED COMPONENTS
+    roscpp
+    std_msgs
+    doa_estimation_msgs
```

Comment:
packages cannot contain custom msgs or srvs files. everything should go to roboy_communication/roboy_communication_cogntion (github repo)

in your code you import from roboy_communication_cognition